Core/include/Acts/Visualization/ViewConfig.hpp

```diff
@@ -49,17 +49,31 @@ struct Color {
   constexpr Color(double r, double g, double b)
       : Color{std::array<double, 3>{r, g, b}} {}
 
+ private:
+  constexpr static int hexToInt(std::string_view hex) {
```

Comment:
Hmm this should be `constexpr`? because `std::stoi(hex, 16)` should also work I think otherwise...

Reply:
Yeah, I'd like this to be `constexpr` if possible. It's not critical, but I think it's nice having the strings (potentially) be converted at compile-time.

Reply:
I'm always hesitant a bit when we reimplement such basic things for not performance critical code, but don't have a very strong opinion on it...

Reply:
This is an internal function, so in the absence of a `constexpr` library function, I think the fallout from reimplementing is not that large.